_data/faq.yml

```diff
@@ -83,23 +82,22 @@
 - question: What are the expectations for mentees in the mentorship programme?
   answer: |
     The expectations for mentees may include setting and achieving goals, being open to feedback and guidance, actively seeking out opportunities to learn, and being committed to their own personal and professional growth. Mentees are expected to actively participate in the mentorship relationship, be open to learning and be willing to share their own experiences and knowledge. <br>
-    <a href="code-of-conduct.html">Please, find more information in Code of Conduct for Mentees</a>
+    <a href="/mentorship-code-of-conduct#mentee">Please, find more information in Code of Conduct for Mentees</a>
```

Comment:
You have to check if the id is correct in the code-of-conduct, please.

Reply:
noted :)

Reply:
Fixed in last commit